developers/weaviate/configuration/replication.md

```diff
@@ -120,6 +120,39 @@ import QueryReplication from '/\_includes/code/replication.get.object.by.id.mdx'
 
 <QueryReplication/>
 
+## Replica movement and status
+
+:::caution Experimental feature
```

Comment:
I wonder if this warning should be stronger & actively discouraged for prod usage.

Reply:
Yep, will rename it to "Do not use in production"